egs/librispeech/s5/local/chain/run_chain_common.sh

```diff
@@ -35,11 +35,11 @@ local/nnet3/run_ivector_common.sh --stage $stage \
 
 
 # Set the variables. These are based on variables set by run_ivector_common.sh
-gmm_dir=exp/tri6b
```

Comment:
I would recommend calling run_ivector_common.sh from the top level script. This would make your run_chain_common.sh script very simple.

For some guidance take a look at  #972 .